### DISCUSSION TITLE
[Web] Cursor position resets to beginning of editor on mobile/tablet browsers

### DISCUSSION THREAD

**Discussion by AndrewJohnson271** [singerdmx/flutter-quill#489] [Web]
After typing for awhile in the editor, when clicking anywhere in the middle of the text, the cursor position resets to the very start of the editor. Once this occurs, it is impossible to enter new text anywhere except the beginning. This issue does not happen when using a browser on a desktop or laptop, but only occurs on mobile or tablet browsers. 

I first experienced this issue within my own app, but have since reproduced this issue using the flutter quill example app. The issue appears to be the exact same in the example app. I have tested the issue on a few iPhones, one android, and a new iPad. I have used multiple browsers for each of these devices and the same issue still arises regardless. 

I am using the latest version of flutter quill, latest version of flutter on the stable channel, and the flutter doctor command results in "no issues found".

**Comment by kapzam123** [comment]
It has been two years since this topic was opened. I'm still having the same issue with mobile browsers. I can't put the cursor where I want in the editor. Please solve this.

**Comment by markodevcic** [comment]
The issue persists still. Is there any workaround?
